src/services/zipRecruiter.jsx

```diff
@@ -0,0 +1,18 @@
+const fetchData = async () => {
+    try {
+      const url = "https://api.ziprecruiter.com/partner/v0/job/emlwcmVjcnVpdGVy-3215";
```

Comment:
Usually, we should call and third-party APIs from our own API. This is especially important if we are using any API keys or tokens with it (which is what I'm assuming the random characters at the end of this URL are) because we don't want them available client-side. You should go ahead and make an endpoint in our API that fetches using this URL and returns the data you need.

Reply:
I actually forgot I had this in my file. I can just remove this since its not needed for now.